test/functional/gridfs_stream.test.js

```diff
@@ -473,6 +473,7 @@ describe('GridFS Stream', function () {
                     // Fail if user tries to abort an aborted stream
                     uploadStream.abort().then(null, function (error) {
                       expect(error.toString()).to.equal(
+                        // TODO(NODE-3405): Replace with MongoStreamClosedError
```

Comment:
You can remove these if they've been resolved in NODE-3405 and this isn't depending on it

Reply:
I think maybe this PR needs to get rebased after things are merged in so that we can keep pulling in latest changes and don't miss anything, let's talk about the order in which we do want to resolve the PRs in stand up